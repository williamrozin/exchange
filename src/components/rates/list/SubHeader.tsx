import { ListSubheader } from '@material-ui/core'
import React, { SFC } from 'react'

interface IProps {
    title: string
}

const SubHeader: SFC<IProps> = (props) =>
    <ListSubheader
        style={ {
            backgroundColor: '#FAFAFA',
            flex: 1,
            padding: 0
        } }>
        { props.title }
    </ListSubheader>

export default SubHeader
