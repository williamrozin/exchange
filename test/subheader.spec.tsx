import { shallow } from 'enzyme'
import React from 'react'
import SubHeader from '../src/components/rates/list/SubHeader'

describe('SubHeader component', () => {
    it('Should render a SubHeader with a title', () => {
        const item = shallow(<SubHeader title='The show must go on' />)
        expect(item.contains('The show must go on')).toBeTruthy()
        expect(item).toMatchSnapshot()
    })
})
