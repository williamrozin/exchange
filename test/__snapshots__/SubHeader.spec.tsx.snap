// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader component should render a SubHeader with a title 1`] = `
<WithStyles(ListSubheader)
  style={
    Object {
      "backgroundColor": "#FAFAFA",
      "padding": "0 18px",
    }
  }
>
  The show must go on
</WithStyles(ListSubheader)>
`;
